src/server/pps/cmds/cmds.go

```diff
@@ -141,12 +96,95 @@ If the job fails, the output commit will not be populated with data.`,
 			return pretty.PrintDetailedJobInfo(os.Stdout, pji)
 		}),
 	}
-	inspectJob.Flags().BoolVarP(&wait, "wait", "w", false, "wait until the job has either succeeded or failed")
 	inspectJob.Flags().AddFlagSet(outputFlags)
-	inspectJob.Flags().AddFlagSet(fullTimestampsFlags)
+	inspectJob.Flags().AddFlagSet(timestampFlags)
 	shell.RegisterCompletionFunc(inspectJob, shell.JobCompletion)
 	commands = append(commands, cmdutil.CreateAlias(inspectJob, "inspect job"))
 
+	waitJob := &cobra.Command{
+		Use:   "{{alias}} <pipeline>@<job>",
+		Short: "Wait for a job to finish then return info about the job.",
+		Long:  "Wait for a job to finish then return info about the job.",
+		Run: cmdutil.RunFixedArgs(1, func(args []string) error {
+			job, err := cmdutil.ParseJob(args[0])
+			if err != nil {
+				return err
+			}
+			client, err := pachdclient.NewOnUserMachine("user")
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+			jobInfo, err := client.WaitJob(job.Pipeline.Name, job.ID, true)
+			if err != nil {
+				errors.Wrap(err, "error from InspectJob")
+			}
+			if raw {
+				return cmdutil.Encoder(output, os.Stdout).EncodeProto(jobInfo)
+			} else if output != "" {
+				return errors.New("cannot set --output (-o) without --raw")
+			}
+			pji := &pretty.PrintableJobInfo{
+				JobInfo:        jobInfo,
+				FullTimestamps: fullTimestamps,
+			}
+			return pretty.PrintDetailedJobInfo(os.Stdout, pji)
+		}),
+	}
+	waitJob.Flags().AddFlagSet(outputFlags)
+	waitJob.Flags().AddFlagSet(timestampFlags)
+	shell.RegisterCompletionFunc(waitJob, shell.JobCompletion)
```

Comment:
I think this name is misleading now; this just completes IDs rather than pipeline@id